src/Workspaces/Remote/ServiceHub/UserOperationBooster.cs

```diff
@@ -31,10 +31,18 @@ public static UserOperationBooster Boost()
             {
                 s_count++;
 
-                if (s_count == 1)
+                try
```

Comment:
I think the `try` may be moved inside the if.

Reply:
Sure, I can switch that around, I wanted a published patch so I could build this locally and see if I need to make any additional changes.